vowpalwabbit/core/src/reductions/cb/details/large_action/one_pass_svd_impl.cc

```diff
@@ -46,6 +46,8 @@ namespace cb_explore_adf
 
 void one_pass_svd_impl::generate_AOmega(const multi_ex& examples, const std::vector<float>& shrink_factors)
 {
+  for (auto* ex : examples) { ex->feature_space_hash = ex->calculate_order_independent_feature_space_hash(); }
```

Comment:
Is this necessary at the beginning?  If `calculate_order_independent_feature_space_hash()` either calculates it or returns the cached value then you can just call it when the hash is needed